src/main/java/core/basesyntax/vehicles/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
Similarly, the stop message should also be informative and indicate that it is an 'Excavator' that has stopped working.